src/logger/loggerApi.c

```diff
@@ -1091,21 +1091,26 @@ static void getTimerConfigs(size_t channelId, void ** baseCfg, ChannelConfig **
     }
 }
 
-static const jsmntok_t * setTimerExtendedField(const jsmntok_t *valueTok, const char *name, const char *value, void *cfg)
+static const jsmntok_t * setTimerExtendedField(const jsmntok_t *valueTok,
+                                               const char *name,
+                                               const char *value,
+                                               void *cfg)
 {
     TimerConfig *timerCfg = (TimerConfig *)cfg;
 
-    int iValue = modp_atoi(value);
-    if (NAME_EQU("st", name))
-        timerCfg->slowTimerEnabled = (iValue != 0);
-    if (NAME_EQU("mode", name))
-        timerCfg->mode = filterTimerMode(iValue);
-    if (NAME_EQU("alpha", name))
+    if (!strcmp("mode", name))
```

Comment:
we should make all of these == 0.  the  not (!) while technically correct, has a higher cognitive load because I think someone will think of it as "not something" first.